Zend/zend_API.c

```diff
@@ -1627,13 +1627,31 @@ ZEND_API void object_properties_load(zend_object *object, HashTable *properties)
 					zend_hash_update(object->properties, key, &tmp);
 				}
 			} else {
+				if (UNEXPECTED(object->ce->ce_flags & ZEND_ACC_NO_DYNAMIC_PROPERTIES)) {
+					zend_throw_error(NULL, "Cannot create dynamic property %s::$%s",
+						ZSTR_VAL(object->ce->name), zend_get_unmangled_property_name(key));
```

Comment:
The call to `zend_unmangle_property_name_ex` above will emit an E_NOTICE (e.g. for `key = "\0invalid"`

This will emit a duplicate notice.

```suggestion
						ZSTR_VAL(object->ce->name), property_info != ZEND_WRONG_PROPERTY_INFO ? zend_get_unmangled_property_name(key): "");
```